package/gluon-wan-dnsmasq/files/etc/init.d/gluon-wan-dnsmasq

```diff
@@ -22,6 +22,7 @@ start_service() {
 		--keep-in-foreground \
 		--pid-file=/var/run/dnsmasq/gluon-wan-dnsmasq.pid \
 		--cache-size=0 \
+		--enable-ubus=gluon-wan-dnsmasq \
 		--resolv-file="$RESOLV_CONF"
 	procd_set_param env LD_PRELOAD=libpacketmark.so LIBPACKETMARK_MARK=1
```

Comment:
Please add a comment that we add `--enable-ubus` purely so that dnsmasq does not drop `CAP_NET_RAW`. And that we need this for libpacketmark.so. I guess it is reasonable to put it right above setting the LD_PRELOAD (which is also pretty close to the `--enable-ubus` anyway).